app/views/scenarios/show.html.haml

```diff
@@ -1,42 +1,11 @@
 .container
-  %h1= @scenario.name
-  - if @scenario.outputs.present?
-    = render 'outputs', variables: @variables
-  %h2 Inputs
-  - @scenario.inputs.each do |key1, value1|
-    - if key1 == 'persons'
-      = render 'persons', people: value1, variables: @variables
-    - elsif key1 == 'families'
-      = render 'families', families: value1, variables: @variables
-    - elsif key1 == 'titled_properties'
-      = render 'properties', properties: value1, variables: @variables
-    - elsif value1.class == Hash || value1.class == Array
-      %hr/
-      %h3.font-italic #{key1.humanize}:
-      - value1.each do |key2, value2|
-        .card
-          .card-body
-            %h4.card-title
-              #{key2.humanize}:
-            - if value2.class == Hash || value2.class == Array
-              - value2.each do |key3, value3|
-                #{key3.humanize}: #{value3}
-            - else
-              = value2
-    - else
-      - vari = @variables.select { |variable| variable['name'] == key1 }[0]
-      %hr/
-      .card
-        .card-body
-          %h6.card-title= link_to vari.description, vari || key
-  %hr/
-  %i
-    %h3 Variables used in this scenario:
-  .card
-    .card-body
-      %ul
-        - @variables.each do |v|
-          %li
-            - if v.namespace.present?
-              %span.badge= v.namespace
-            = link_to v.description, v
+  %h1.display-3= @scenario.name
```

Comment:
@Br3nda what does `.display-3` do? Also on percy the Families heading needs margin above it.

Reply:
Display makes it bigger. It's part of bootstrap.